wiki/Manual.md

```diff
@@ -4136,61 +4136,55 @@ exchange.create_limit_sell_order (symbol, amount, price[, params])
 
 #### Stop Orders
 
-Stop orders, are placed onto the order book when the price of the underlying asset reaches the trigger price.
+Coming from traditional trading, the term "Stop" order has been a bit ambigious, so instead of it, in CCXT we use term "Trigger" order. When symbol's price reaches your "trigger"("stop") price, the order is activated - if you had chosen *market* order, then it will be executed immediately, if you have chosen limit order, it will be placed in the orderbook.
+
+We have different classification of trigger orders:
+* stand-alone [Trigger order](#trigger-orders) to buy/sell coin (open/close position)
+* stand-alone [Stop-Loss](#stop-loss-orders) or [Take-Profit](#take-profit-orders) order which are only designed to close an open position.
+* an attached Stop-Loss or Take-Profit order into a primary order ([Conditional Trigger Order](#stopLoss-and-takeProfit-orders-attached-to-a-position)).
 
-* They can be used to close positions when a certain profit level is reached, or to mitigate a large loss.
-* They can be stand-alone orders ([Trigger](#trigger-order), [Stop Loss](#stop-loss-orders), [Take Profit](#take-profit-orders)).
-* Or they can be attached to a primary order ([Conditional Stop Orders](#stopLoss-and-takeProfit-orders-attached-to-a-position)).
-* Stop Orders can be limit or market orders
 
 ##### Trigger Orders
 
-Traditional "stop" order (which you might see across exchanges' websites) is now called "trigger" order across CCXT library. Implemented by adding a `triggerPrice` parameter. They are independent basic trigger orders that can open and close a position.
+Traditional "stop" order (which you might see across exchanges' websites) is now called "trigger" order across CCXT library. Implemented by adding a `triggerPrice` parameter. They are independent basic trigger orders that can open or close a position.
 
-* Activated when price of the underlying asset/contract crosses the `triggerPrice` from **any direction**
+* Typically, it is activated when price of the underlying asset/contract crosses the `triggerPrice` from **any direction**. However, some exchanges' API require to set `triggerDirection` too, which triggers order depending whether price is above or below `triggerPrice`. For example, if you want to trigger  limit order (buy 0.1 `ETH` at limit price `1500`) once pair price crosses `1700`:
 
 ```javascript
 // JavaScript
-const symbol = 'ETH/BTC'
-const type = 'limit' // or 'market'
-const side = 'sell'
-const amount = 123.45 // your amount
-const price = 54.321 // your price
 const params = {
-    'triggerPrice': 123.45, // your stop price
+    'triggerPrice': 1700,
 }
-const order = await exchange.createOrder (symbol, type, side, amount, price, params)
+const order = await exchange.createOrder ('ETH/USDT', 'market', 'buy', 0.1, 1500, params)
```

Comment:
imo there were bloated lines. it's really odd to separate lines here to let know users argument names of `createOrder`. readers here already know least ccxt basics